### DISCUSSION TITLE
The result of running code src.crs did not match the attributes shown in QGIS and an `ProjError: Error creating Transformer from CRS.` was occurred.

### DISCUSSION THREAD

**Discussion by ghost** [giswqs/geog-312#121] [Q&A] [answered]
Hi, Prof. Wu. I am learning rasterio.ipynb. I find that running the code src.crs results in `CRS.from_wkt('LOCAL_CS["WGS 84 / Pseudo-Mercator",UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH]]')` instead of `EPSG:3857 - WGS 84 / Pseudo-Mercator` in QGIS. As a result, I get an error `ProjError: Error creating Transformer from CRS.` when I run the code `gdf = gdf.to_crs(src.crs)`. How do I fix this error? Thank you!

**Comment by giswqs** [comment, marked as answer]
See https://github.com/giswqs/geog-312/discussions/113#discussioncomment-10962924

**Reply by ghost** [reply]
Thank you. But the return value of `src.crs.to_epsg()` is None.  I'm not sure why.
![epsg](https://github.com/user-attachments/assets/c4d9799b-1be5-411f-a7f8-58db6f6b4060)
